project/src/utils.js

```diff
@@ -46,21 +46,70 @@ const mapOfferToClient = (offer) => ({
   goods: offer.goods,
   hostAvatarUrl: offer.host.avatar_url,
   host: offer.host,
-  hostId:offer.host.id,
+  hostId: offer.host.id,
   hostName: offer.host.name,
   hostIsPro: offer.host.is_pro,
   description: offer.description,
-  city:offer.city,
-  cityName:offer.city.name,
-  cityLocation:offer.city.location,
-  cityLatitude:offer.city.location.latitude,
-  cityLongitude:offer.city.location.longitude,
-  cityZoom:offer.city.location.zoom,
-  location:offer.location,
-  latitude:offer.location.latitude,
-  longitude:offer.location.longitude,
-  zoom:offer.location.zoom,
+  city: offer.city,
+  cityName: offer.city.name,
+  cityLocation: offer.city.location,
+  cityLatitude: offer.city.location.latitude,
+  cityLongitude: offer.city.location.longitude,
```

Comment:
```suggestion
  city: {
    name: offer.city.name,
    location: {
      // и тд
    },
  },
```
Вкладывать объекты тоже можно)